ocp_resources/user_defined_network.py

```diff
@@ -66,3 +67,65 @@ def to_dict(self) -> None:
                 _spec["localNet"] = self.local_net
 
     # End of generated code
+
+
+LAYER2: str = "Layer2"
```

Comment:
Why did this need to be a global constant?

Reply:
Right, done.